docs/src/examples/components/SplitButton/Types/SplitButtonExample.shorthand.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { Flex, SplitButton } from '@stardust-ui/react'
+
+class SplitButtonExampleShorthand extends React.Component {
+  items = [
+    {
+      key: 'conversation',
+      content: 'New conversation',
+    },
+    {
+      key: 'group',
+      content: 'New group message',
+    },
+    {
+      key: 'channel',
+      content: 'New channel message',
+    },
+  ]
+  state = {
+    index: 0,
+    message: 'Send a message',
+  }
+
+  handleClick = () => {
+    this.setState({ message: 'Great! Now start writing your message.' })
+  }
+
+  render() {
+    const { index, message } = this.state
+    const filteredItems = [...this.items.slice(0, index), ...this.items.slice(index + 1)]
+    return (
+      <>
+        <div>{message}</div>
+        <Flex gap="gap.small">
+          <SplitButton
+            onMainButtonClick={this.handleClick}
+            onMenuItemClick={this.handleClick}
+            menu={filteredItems}
+            button={this.items[index]}
+          />
+          <SplitButton
+            onMainButtonClick={this.handleClick}
+            onMenuItemClick={this.handleClick}
+            menu={filteredItems}
+            button={this.items[index]}
+            primary
+          />
+        </Flex>
+      </>
+    )
+  }
+}
+
+export default SplitButtonExampleShorthand
```

Comment:
This example is not functional. I suggest to separate it to two examples to keep it simple:

### `SplitButtonExample.shorthand.tsx`
```tsx
import * as React from 'react'
import { Flex, SplitButton } from '@stardust-ui/react'

const SplitButtonExampleShorthand = () => (
  <SplitButton
    menu={[
      { key: 'group', content: 'New group message' },
      { key: 'channel', content: 'New channel message' },
    ]}
    button="New conversation"
  />
)

export default SplitButtonExampleShorthand
```

### `SplitButtonExamplePrimary.shorthand.tsx` (under `Types`)
```tsx
import * as React from 'react'
import { Flex, SplitButton } from '@stardust-ui/react'

const SplitButtonExamplePrimaryShorthand = () => (
  <SplitButton
    menu={[
      { key: 'group', content: 'New group message' },
      { key: 'channel', content: 'New channel message' },
    ]}
    button="New conversation"
    primary
  />
)

export default SplitButtonExamplePrimaryShorthand
```

### `SplitButtonExampleMenu.shorthand.tsx` (under `Usage`)

```tsx
import { useNumberKnob } from '@stardust-ui/docs-components'
import { Flex, SplitButton } from '@stardust-ui/react'
import * as React from 'react'

const items = [
  { key: 'group', content: 'New group message' },
  { key: 'channel', content: 'New channel message' },
  { key: 'conversation', content: 'New conversation' },
]

const SplitButtonExampleMenuShorthand = () => {
  const [activeIndex, setActiveIndex] = useNumberKnob({ name: 'activeIndex' })
  const activeItem = items[activeIndex]

  return (
    <SplitButton
      button={activeItem}
      menu={{
        activeIndex,
        items,
      }}
      onMenuItemClick={(e, { index }) => setActiveIndex(index)}
    />
  )
}

export default SplitButtonExampleMenuShorthand
```